src/controllers/confirmation.controller.ts

```diff
@@ -21,7 +21,7 @@ export const get = (req: Request, res: Response) => {
     referenceNumber,
     entityEmail: appData.entity?.email,
     userEmail: getLoggedInUserEmail(req.session),
-    workingDays: 2,
+    workingDays: 14,
```

Comment:
i'd change workingDays to be verificationCheckDays as it looks like it is no longer 'working' days but just days. Also need to update the matching variable in the template

Reply:
Sure